src/components/PopupWithForm.js

```diff
@@ -16,6 +17,14 @@ export default class PopupWithForm extends Popup {
     return inputValues;
   }
 
+  setLoading(isLoading) {
```

Comment:
Great that you've implemented the logic for rendering loading via the class methods